Cargo.toml

```diff
@@ -118,3 +118,5 @@ gdk-pixbuf = { version = "0.15", features = [ "v2_36_8" ] }
 libayatana-appindicator = { version = "0.2", optional = true }
 libappindicator = { version = "0.7", optional = true }
 x11-dl = "2.18"
+uuid = { version = "0.8", features = [ "v4" ] }
+image = "0.24"
```

Comment:
```suggestion
image = { version = "0.24", default-features = false, features = [ "png" ] }
```

Reply:
I don't think we actually need any features for release. `png` is only needed for reading the `.png` image in the examples.

Reply:
Yep, you're right :+1: